src/zenml/secrets_manager/__init__.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""
+## Secret Manager
+
+...
+"""
+from zenml.integrations.aws.secrets_manager import AWSSecretsManager
+from zenml.secrets_manager.base_secrets_manager import BaseSecretsManager
+from zenml.secrets_manager.local.local_secrets_manager import (
+    LocalSecretsManager,
+)
+
+__all__ = ["BaseSecretsManager", "LocalSecretsManager", "AWSSecretsManager"]
```

Comment:
Just going by what we've implemented with other stack components, I think we shouldn't be importing the `AWSSecretsManager` here.